content/courses/tokens-and-nfts/token-program-advanced.md

```diff
@@ -12,7 +12,8 @@ description:
 
 Finally, we'll cover burning tokens, and delegation. You may not use these in
 your own application, so if you're really excited about NFTs, feel free to skip
-to [creating NFTs with Metaplex](/content/courses/tokens/nfts-with-metaplex)!
+to
+[creating NFTs with Metaplex](/content/courses/tokens-and-nfts/nfts-with-metaplex)!
```

Comment:
```suggestion
[creating NFTs with Metaplex](/content/courses/tokens-and-nfts/nfts-with-metaplex.md)!
```

Thanks! We do links in a slightly odd way, see https://github.com/solana-foundation/developer-content/blob/main/CONTRIBUTING.md#links

Please go through and add the extensions on all the links - I know it's odd!